content/en/tracing/profiling/search_profiles.md

```diff
@@ -0,0 +1,138 @@
+| Lock                     | Shows the time each method spent waiting for a lock.                                                                                                                                                                                                                                               |
+| Socket I/O               | Shows the time each method spent handling socket I/O.                                                                                                                                                                                                                                              |
+
+
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+Once enabled, the following profile types are collected:
+
+| Profile type             | Definition                                                                                                                                                                                                                                                                                         |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU         | Shows the time each function spent running on the CPU. It includes CPython bytecode, including native code called from within Python.                                                                                                                                                                     |
+| Allocation               | Shows the amount of heap memory allocated by each function, including allocations which were subsequently freed. Only supported on Python 3.                                                                                                                                                                                    |
+| Wall | Shows the elapsed time used by each function. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function is running. |
+| Exceptions               | Shows the number of caught or uncaught exceptions raised by each function.                                                                                                                                                                                                                                                 |
+| Lock                     | Shows the time each function spent waiting for a lock.                                                                                                                                                                                                                                               |
```

Comment:
A lot of these descriptions focus only on the default summary table function. But we can actually show more stuff. For example, with the merging of the 4 python locks, the updated description is:

> Shows the time each function spent in locking (waiting for or holding a lock) or the number of times a function was observed locking/unlocking a lock.

https://github.com/DataDog/web-ui/blob/c02849b22d57ec26ed7f562573ba12c822ece1cc/javascript/datadog/profiling/lib/profile-types-definitions.ts#L334-L335